src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -61,7 +61,8 @@
 ASAN_REGEX = re.compile(r'.*(ERROR: AddressSanitizer)[: ]*' r'[ ]*([^(:]+)')
 ASSERT_REGEX = re.compile(
     r'(?:\[.*?\]|.*\.(?:%s):.*)?' % ('|'.join(C_CPP_EXTENSIONS)) +
-    r'\s*(?:ASSERT(?:ION)? FAIL(?:URE|ED)|panic): (.*)', re.IGNORECASE)
+    r'\s*(?:ASSERT(?:ION)? FAIL(?:URE|ED)|panic|fatal error): (.*)',
```

Comment:
Fatal error is not an assertion failure. i looked at some examples like 
https://stackoverflow.com/questions/37551288/golang-fatal-error-runtime-out-of-memory, so this feels like it should be seperate.
Maybe just change change our current fatal error regex. It should then create the right crash type 'Fatal error'
FATAL_ERROR_REGEX = re.compile(r'#\s*Fatal error(:| in) (.*)')

Reply:
Yeah, that felt weird, but I didn't notice the fatal error regex. Thanks for the suggestion, done.